### PR TITLE
Revert Style/ArgumentsForwarding Rubocop rule

### PR DESCRIPTION
**Pull request by andreibondarev**

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -1,6 +1,9 @@
 AllCops:
   SuggestExtensions: false
 
+Style/ArgumentsForwarding:
```
Comment (Rukomoynikov) [inline .rubocop.yml:4]: I can be wrong, but could you please try to move `Style/ArgumentsForwarding` below all other instructions in this file?

Comment (Rukomoynikov) [reply .rubocop.yml:4]: As they can overwrite the instructions above, including `Style/ArgumentsForwarding`

Comment (Rukomoynikov) [reply .rubocop.yml:4]: It didn't help. I tried. :)

Comment (andreibondarev) [reply .rubocop.yml:4]: @Rukomoynikov It's very odd. I don't quite understand why these rules are being ignored in `.rubocop.yml` or `.standard.yml`.

**Comment by andreibondarev** [comment]
@Rukomoynikov Looks like you fixed it! 👀 🎉

**Comment by Rukomoynikov** [comment]
> @Rukomoynikov Looks like you fixed it! 👀 🎉

Thank you, but I just slightly altered your solution.

**Pull request merged**